test/redis.test.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+if (process.env.CI) {
```

Comment:
why have you introduced this `if` here? If we are running tests for Redis, let's run them locally too.

Reply:
You can run unit tests with redis by doing npm run test:ci 

I just dont want to force a dev, who is using windows, to go to the hassle to activate WSL2 to be able to contribute. 

Also we have alot of flushall calls, so it has side effects. I dont want that somebody later complains, that by running the unit tests in his docker container, which is in his test or prod environment, and having a redis bound to localhost to be accidently flushed, because he tested the unit tests shipped with the npm-module.

Reply:
If you insist, then I will change. Then I would prefer to introduce a NO_REDIS env variable to disable the unit tests, if needed.

Reply:
created a npm script test:unit:memory with NO_REDIS env flag.